src/app/v2/page.tsx

```diff
@@ -1,13 +1,17 @@
 'use client';
 
-import { Text } from '@penumbra-zone/ui/Text';
+import { useEffect } from 'react';
+import { useRouter } from 'next/navigation';
+import { PagePath } from '@/utils/routes/pages.ts';
 
-const HomePage = () => {
-  return (
-    <section>
-      <Text h2>Hi!</Text>
-    </section>
-  )
-}
+const RedirectPage = () => {
+  const router = useRouter();
 
-export default HomePage;
+  useEffect(() => {
```

Comment:
issue: this will only redirect when the page is loaded. Instead, use `redirect` function that will re-route on the server side: https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating#redirect-function